frontend/ee/components/UsersPage/UsersTable.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { Pagination } from '@/_components';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+const UsersTable = ({
+  isLoading,
+  users,
+  archivingUser,
+  unarchivingUser,
+  meta,
+  generateInvitationURL,
+  invitationLinkCopyHandler,
+  unarchiveOrgUser,
+  archiveOrgUser,
+  pageChanged,
+  darkMode,
+}) => {
+  const tableRef = React.createRef(null);
+
+  function calculateOffset() {
+    const elementHeight = tableRef.current.getBoundingClientRect().top;
+    return window.innerHeight - elementHeight;
+  }
+
+  const setWidth = () =>
```

Comment:
I don't understand the reason to setWidth on resize event, can you once please elaborate

Reply:
Otherwise, if the user resizes the window, the pagination wrapper will not responsive.